src/utils/flow.js

```diff
@@ -112,15 +112,25 @@ export function objectValues<Value, Obj: {| [string]: Value |}>(
 }
 
 /**
- * This is a type-friendly version of Object.entries that assumes the object has
- * a Map-like structure.
+ * This is a type-friendly version of Object.entries that assumes the object is an
+ * exact object.
  */
-export function objectEntries<Value, Obj: {| [string]: Value |}>(
+export function objectEntriesExact<Value, Obj: {| [string]: Value |}>(
   object: Obj
 ): Array<[string, Value]> {
   return (Object.entries: Function)(object);
 }
 
+/**
+ * This is a type-friendly version of Object.entries that assumes the object has
+ * a Map-like structure.
+ */
+export function objectEntriesMap<Key, Value>(object: {
+  [Key]: Value,
+}): Array<[Key, Value]> {
+  return (Object.entries: Function)(object);
+}
+
```

Comment:
In the context of these functions, I wonder what additional safety `objectEntriesExact` brings over `objectEntriesMap`. I'd keep only the latter.

Reply:
Yeah, it looks like I don't need to preserve `objectEntriesExact`. I think I was just trying not to touch that other code's implementation. The types are doing the equivalent thing using `objectEntriesMap` everywhere.